Sources/Paywalls/Components/PaywallPurchaseButtonComponent.swift

```diff
@@ -14,17 +14,41 @@ public extension PaywallComponent {
 
     struct PurchaseButtonComponent: PaywallComponentBase {
 
-        let type: String
-        let cta: PaywallComponent.LocalizationKey
-        let ctaIntroOffer: PaywallComponent.LocalizationKey?
+        let type: ComponentType
+        public let ctaLid: LocalizationKey
+        public let ctaIntroOfferLid: LocalizationKey
+        public let fontFamily: String?
+        public let fontWeight: FontWeight
+        public let color: ColorInfo
+        public let textStyle: TextStyle
+        public let horizontalAlignment: HorizontalAlignment
+        public let backgroundColor: ColorInfo?
+        public let padding: Padding
+        public let margin: Padding
```

Comment:
I'm not up to date with the product specs so please ignore this comment unless I say something useful 😵.  I was wondering if we were planning to support some features like:
- Gradient colors
- Background images
- Shadow/elevation (in android)
- Leading/trailing icons
- ...

Maybe they are planned for later, which is fine, just asking.

Reply:
Yeah yeah, lots of these are planned for later 😁